ignite/handlers/param_scheduler.py

```diff
@@ -792,6 +792,69 @@ def simulate_values(  # type: ignore[override]
             return output
 
 
+class _CosineAnnealingWarmRestarts:
+    def __init__(self, lr_scheduler: torch.optim.lr_scheduler.CosineAnnealingWarmRestarts):
```

Comment:
Let's directly import `CosineAnnealingWarmRestarts` in the begining of the file to avoid long names like `torch.optim.lr_scheduler.CosineAnnealingWarmRestarts`